src/MetricExporter.ts

```diff
@@ -117,11 +120,11 @@ export class DynatraceMetricExporter implements MetricExporter {
 					// TODO: when metrics 0.20.0 is released use aggregator.aggregationTemporality
 					const data = metric.aggregator.toPoint();
 					const normalizedMetric = this._dtMetricFactory
-						.createDeltaCounter(
+						.createCounterDelta(
 							metric.descriptor.name,
 							dimensions,
 							data.value,
-							hrTimeToMilliseconds(data.timestamp)
+							new Date(hrTimeToMilliseconds(data.timestamp))
```

Comment:
Is there not already a hr time to date function?

Reply:
No, it offers `toNanos`, `toMicros`, `toMillis`.